packages/pixeloven-docs/src/configs/config.tsx

```diff
@@ -30,6 +31,20 @@ addDecorator(
 );
 addDecorator(withKnobs);
 
+
+/**
+ * Wrapper with custom id for modals to attach to
+ */
+const modalContainerDecorator = (storyFn: () => {}) => (
+    <>
```

Comment:
Never seen empty tags before, is this like `React.Fragment`?

Reply:
Yeah!! It’s react fragment! I have some of this in our project too, it’s very shorthand. Tsx supports :)